lib/utils/fs/fileExistsSync.test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
```

Comment:
This whole file can be removed once `this.serverless.utils.fileExistsSync` is used.